api/proto/teleport/scim/v1/scim_service.proto

```diff
@@ -44,6 +44,10 @@ service SCIMService {
 
 // ListSCIMResourcesRequest represents a request to fetch multiple resources
 message ListSCIMResourcesRequest {
+  // Meta holds information about the original SCIM/HTTP request that triggered
+  // this internal SCIM operation request.
+  RequestMetadata meta = 4;
```

Comment:
Can we keep the fields in numerical order instead of adding this new field at the top of each message?